tests/zfs-tests/include/tunables.cfg

```diff
@@ -72,6 +72,8 @@ MULTIHOST_INTERVAL		multihost.interval		zfs_multihost_interval
 OVERRIDE_ESTIMATE_RECORDSIZE	send.override_estimate_recordsize	zfs_override_estimate_recordsize
 PREFETCH_DISABLE		prefetch.disable		zfs_prefetch_disable
 RAIDZ_EXPAND_MAX_REFLOW_BYTES	vdev.expand_max_reflow_bytes	raidz_expand_max_reflow_bytes
+READ_SIT_OUT_SECS		vdev.read_sit_out_secs		vdev_read_sit_out_secs
+SIT_OUT_CHECK_INTERVAL		vdev.raidz_outlier_check_interval_ms	vdev_raidz_outlier_check_interval_ms
```

Comment:
`vdev_raidz_outlier_insensitivity` should be added.

Reply:
Do we need to add that even though it isn't actually used in any tests? I've never been entirely clear on that.

Reply:
You're right, it's not functionally needed for anything.  I think it's just nice to have a complete list of the tunables here which could be used in the test cases.  That said, I'm sure more than a few are still missing.